web/src/components/people/ApprovedResearcherImport.tsx

```diff
@@ -0,0 +1,43 @@
+import Button from "@/components/ui/Button";
+import { postPeopleApprovedResearchersImportCsv } from "@/openapi";
+import { useRef, useState } from "react";
+
+export default function ApprovedResearcherImport() {
+  const inputRef = useRef<HTMLInputElement>(null);
+  const [buttonText, setButtonText] = useState("Import approved researchers");
+  const [buttonDisabled, setButtonDisabled] = useState(false);
+
+  async function handleFileUpload(e: React.ChangeEvent<HTMLInputElement>) {
+    const files = e.target.files;
+    if (!files) return;
+
+    const response = await postPeopleApprovedResearchersImportCsv({
+      body: files[0],
+    });
+    if (response.error) {
+      console.log(response.error);
```

Comment:
```suggestion
      console.error(response.error);
```

Reply:
the errors aren't user actionable so I'm inclined to not. if this was going to be used a lot then we'd send helpful error messages back from the API..

Reply:
Usually I would also be inclined to make this a bit more user friendly. For instance, the button becomes disabled on error and requires a manual refresh. But I guess this upload feature will become obsolete as soon as Sharepoint is replaced, so perhaps it's not worth adding additional features?